components/fathom/sources/common/base-webhook.mjs

```diff
@@ -0,0 +1,87 @@
+import fathom from "../../fathom.app.mjs";
+import { ConfigurationError } from "@pipedream/platform";
+
+export default {
+  props: {
+    fathom,
+    db: "$.service.db",
+    http: "$.interface.http",
+    includeActionItems: {
+      propDefinition: [
+        fathom,
+        "includeActionItems",
+      ],
+    },
+    includeCrmMatches: {
+      propDefinition: [
+        fathom,
+        "includeCrmMatches",
+      ],
+    },
+    includeSummary: {
+      propDefinition: [
+        fathom,
+        "includeSummary",
+      ],
+    },
+    includeTranscript: {
+      propDefinition: [
+        fathom,
+        "includeTranscript",
+      ],
+    },
```

Comment:
Optional suggestion: these four props could be transformed into a required `string[]` prop instead, so the user can select one or more options to be included - then the code could check each of them with `.includes`

This would eliminate the possibility of activating the source with none of them selected (handled by the ConfigurationError below)